tasks/config/run.js

```diff
@@ -25,13 +25,18 @@ module.exports = function (grunt) {
     return flags;
   }, []);
 
+  const devEnv = Object.assign(process.env, {
+    NODE_OPTIONS: '--max-old-space-size=1024'
```

Comment:
Yeah, I am over writing the --max-old-space-size which is set in bin/kibana for these tasks.